synapse/lib/stormtypes.py

```diff
@@ -3526,6 +3585,7 @@ def getObjLocals(self):
             'open': self._methTeleOpen,
         }
 
+    @stormfunc(readonly=True)
```

Comment:
Should we remove it from `$lib.service.get` as well or is it worth keeping for the permission/existence checking?

Reply:
I don't think its worth keeping around for `$lib.service.get()` since the object returned is functionally useless in a readonly runtime. I'll remove it.